CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+# Contributing
+
+## Making a new release
+
+1. Run `git-chglog --next-tag vX.Y.Z --output CHANGELOG.md`.
+2. Format if needed, commit and open a PR (as `main` branch is protected).
+3. Wait for the PR to be merged.
+4. Once the PR is in main, create an annotated signed tag on main HEAD:
```

Comment:
Not to be picky, but between 3 and 4, if the PR takes some time to merge and if the author changes its current position in history, step 4 will tag something else. Can we pinpoint what commit needs to be tagged instead of tagging HEAD?

Reply:
tootally fair. Done.